sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1031,14 +1031,14 @@ def ReadItems(
         self,
         collection_link: str,
         feed_options: Optional[Mapping[str, Any]] = None,
-        response_hook: Optional[Callable[[Mapping[str, Any], Mapping[str, Any]], None]] = None,
+        response_hook: Optional[Callable[[CaseInsensitiveDict, Dict[str, Any]], None]] = None,
         **kwargs: Any
     ) -> ItemPaged[Dict[str, Any]]:
         """Reads all documents in a collection.
         :param str collection_link: The link to the document collection.
         :param dict feed_options: The additional options for the operation.
         :param response_hook: A callable invoked with the response metadata.
-        :type response_hook: Callable[[Dict[str, str], Dict[str, Any]]
+        :type response_hook: Callable[[CaseInsensitiveDict, Dict[str, Any]]
```

Comment:
I think some of these callables are missing the return type? Or if the return type is missing does mypy automatically assume None?
`Callable[[Dict[str, str], Dict[str, Any], None]`